src/main/kotlin/it/pagopa/wallet/controllers/WalletController.kt

```diff
@@ -147,4 +169,15 @@ class WalletController(
             }
         }
     }
+
+    private fun getAuthenticationToken(exchange: ServerWebExchange): Mono<String> {
+        return Mono.justOrEmpty(
+            Optional.ofNullable(exchange.request.headers["Authorization"])
```

Comment:
```suggestion
            Optional.ofNullable(exchange.request.headers[HttpHeaders.AUTHORIZATION])
```
please use constant from ` org.springframework.http.HttpHeaders` instead of hardcoded one

Reply:
Fixed with 60f1613c21a49e7a9e967f931c04fe6a1bc0331d